R/theme-defaults.r

```diff
@@ -93,7 +93,10 @@ theme_grey <- function(base_size = 11, base_family = "") {
                          ),
 
     legend.background =  element_rect(colour = NA),
-    legend.margin =      unit(0.2, "cm"),
+    legend.spacing =     unit(0.4, "cm"),
+    legend.hspacing =    NULL,
```

Comment:
That is up to you - the legend.margin used this approach so I just ported it, but for consistency we should probably change it...